Week5/Week5_Exercise1.js

```diff
@@ -0,0 +1,18 @@
+let count = 0;
+
+function getDate() {
+    setInterval(function() {document.getElementById("date").innerHTML=Date()}, 1000) 
+}
+
+function countUp() {
+    count = count + 1;
+    document.getElementById("counter").innerHTML = count; 
+    console.log("Count is ", count)
```

Comment:
When passing multiple arguments to `console.log`, they are automatically separated by a space.

Reply:
That's weird! I'm not sure I like that! But thanks for pointing it out.

Reply:
Yeah, I don't like it either. I've gotten used to it, but I still have to go back and remove spaces before `",` almost every time! What ever happened to `printf`? :smiling_face_with_tear: